src/casp/ml/data/transforms.py

```diff
@@ -0,0 +1,57 @@
+import typing as t
+
+import torch
+
+from casp.ml.running_stats import OnePassMeanVarStd
+from casp.ml.utils import PickleMixin
+
+
+class CASTransform(PickleMixin):
+    def __call__(self, tensor: torch.Tensor) -> torch.Tensor:
+        raise NotImplementedError
+
+
+class Compose(CASTransform):
+    """
+    Inspired by `torchvision.transforms.Compose`
+    """
+
+    def __init__(self, transforms: t.List[CASTransform]):
+        self.transforms = transforms
+
+    def __call__(self, tensor: torch.Tensor) -> torch.Tensor:
+        for transform in self.transforms:
+            tensor = transform(tensor)
+        return tensor
+
+    def __repr__(self) -> str:
+        format_string = f"{self.__class__.__name__}("
+        for transform in self.transforms:
+            format_string += "\n"
+            format_string += f"    {transform}"
+
+        format_string += "\n)"
+        return format_string
+
+
+class RowWiseNormalization(CASTransform):
+    def __init__(self, sc_rna_normalization_pseudocount: int):
```

Comment:
I believe it is called so because this is a factor that they use in the formula for normalizing scRNA data. Like it's scRNA normalization-specific variable that makes it different from other techniques that are not used in scRNA data. It was Mehrtash's recommendation to call this variable like this. I am sure he can explain the logic behind it better than me.)